tests/avalon_test_framework/tests/work_order_tests/test_submit.py

```diff
@@ -76,7 +73,6 @@ def test_workordersubmit_datahash_null(self):
                 result_response,
                 "Invalid data format for data hash of in data")
             is ResultStatus.SUCCESS.value)
-        logger.info('\t\t!!! Test completed !!!\n\n')
```

Comment:
Removed all logger.info Test completed messages as they are now part of teardown